CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ### Added
 - Chromecast support 
 - Update content metadata attributes on the receiver directly via the sender instance of the integration
+- Changelog entry about NPM usage
```

Comment:
Are you normally writing `CHANGELOG` or `Changelog`?

And is this best located in the `Added` section? For me that's more an internal thing.

Reply:
-> `changelog`

We don't have an internal section as this is open source.